components/slack/sources/common/base.mjs

```diff
@@ -1,21 +1,77 @@
 import slack from "../../slack.app.mjs";
+import {
+  NAME_CACHE_MAX_SIZE, NAME_CACHE_TIMEOUT,
+} from "./constants.mjs";
 
 export default {
   props: {
     slack,
-    nameCache: "$.service.db",
+    db: "$.service.db",
   },
   methods: {
-    async maybeCached(key, refreshVal, timeoutMs = 3600000) {
-      let record = this.nameCache.get(key);
+    cleanCache(cacheObj) {
+      console.log("Initiating cache check-up routine...");
+      const timeout = Date.now() - NAME_CACHE_TIMEOUT;
+
+      const entries = Object.entries(cacheObj);
+      let cleanArr = entries.filter(
+        ([
+          , { ts },
+        ]) => ts > timeout,
+      );
+      const diff = entries.length - cleanArr.length;
+      if (diff) {
+        console.log(`Cleaned up ${diff} outdated cache entries.`);
```

Comment:
```suggestion
        console.log(`Cleaned up ${diff} outdated cache entries.`);
```

Reply:
Removed period at the end of the log message (Google's Developer docs style guide: Punctuation)